docs/src/main/paradox/scala/http/routing-dsl/httpApp.md

```diff
@@ -0,0 +1,71 @@
+<a id="http-app"></a>
+# HttpApp Bootstrap
+
+@@@ warning { title="API may change" }
+This is experimental and the API is subjected to change in futures releases of Akka HTTP.
+For further information about this marker, see [The @DoNotInherit and @ApiMayChange markers](http://doc.akka.io/docs/akka/current/common/binary-compatibility-rules.html#The_@DoNotInherit_and_@ApiMayChange_markers)
+in the Akka documentation.
+@@@
+
+@@toc { depth=1 }
+
+## Introduction
+
+The objective of `HttpApp` is to quickly get you started starting a Http Server with just a few lines of code.
```

Comment:
Nitpicking to remove `started starting`: ... is to help you start an HTTP server with just ...